docs/pages/ci.md

```diff
@@ -15,19 +15,21 @@ layout: default
 
 <details>
 <summary> 🟢 explanation</summary>
-We have many projects using github CI, it's both convenient and works well. It's become the standard. Also, note that it's free for public repos.
+We have many projects using GitHub CI and, it has good integration with GitHub itself, and is free for public repositories (with limited free monthly minutes for private repositories).
 </details>
 
 # Coverage monitoring
 
-You can check coverage simply in the terminal with the
-[pytest-cov](https://pypi.org/project/pytest-cov/) plugin. However you might
-consider a third-party code coverage analytics and reporting service. There are
-two that we've used and they're both rather similar. They render the code with
-highlighting to show which lines are not executed but the tests, and can track
-test coverage over time.
+These services report and track test code coverage over time.
+They render the code with highlighting to show which lines are not executed by tests.
+See [testing](testing.md) for our recommendations on packages to generate code coverage during tests.
```

Comment:
I claim Jekyll is not smart enough to make this a link.
```
[2024-03-25 17:27:17] ERROR `/pages/testing.md' not found.
```

Reply:
For some reason I thought we were using `mkdocs` so I went and looked up how to link in the mkdocs docs... 🤦

Reply:
Hold on how did you work this out? All the CI is 🟢 , did you do a local build? CI definitely shouldn't be 🟢 if the docs won't build...

Reply:
Yeah I just did a `bundle exec jekyll serve` locally... Actually, it's because I would've been impressed that links were replaced like that.

Which might mean I'm impressed with `MkDocs`...? 🤣

Reply:
I guess our CI doesn't complain because the `jekyll`/JTD build doesn't do any link-checking/

Reply:
There's `bundle exec jekyll build  --strict_front_matter` but that doesn't seem to fail here.

Reply:
```suggestion
See [testing](testing) for our recommendations on packages to generate code coverage during tests.
```

Seems to work!